### ISSUE TITLE
How to update front page text

### ISSUE THREAD

**Issue by rmzelle** [Zotero-ODF-Scan/zotero-odf-scan#20]
@adam3smith, I have a few suggestions for the text at https://zotero-odf-scan.github.io/zotero-odf-scan/. What is the best way to do this? Just directly create a PR for https://github.com/Zotero-ODF-Scan/zotero-odf-scan/blob/gh-pages/index.html?

And how does that file relate to https://github.com/Zotero-ODF-Scan/zotero-odf-scan/blob/gh-pages/documentation/index.html?

**Comment by adam3smith** [comment]
PR to gh-pages works great, yes. I had completely forgotten that the documentation folder even exists. It's an older version that could as well get deleted.

**Comment by rmzelle** [comment]
And no immediate plans to go back to a markdown source document?

**Comment by adam3smith** [comment]
No. I forgot the details, but there were a bunch of things I couldn't get the .md workflow to do, so I stuck to writing this in straight html.
